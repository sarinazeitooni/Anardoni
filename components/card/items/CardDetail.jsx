import React from "react";
import style from '../scss/Card.module.scss';
import Image from 'next/image';
const CardDetail=({detailImage,title,subTitle,id})=>{
    return(
        <div className={style['card-detail-container']}>
            <div className={style['card-detail-text']}>
                <h4>{title}</h4>
                <p>{subTitle}</p>
            </div>
            <div className={style['card-detail-image']}>
                <Image src={detailImage} width={76} alt='card-detail-image' height={76}/>
            </div>
        </div>
    )
}
export default CardDetail;