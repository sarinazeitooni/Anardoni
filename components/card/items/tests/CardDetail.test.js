import React from 'react';
import { shallow } from 'enzyme';
import CardDetail from "../CardDetail";
describe("CardDetail component" , ()=>{
    const container = shallow(<CardDetail/>);
    it("it should render a div",()=>{
        expect(container).toMatchSnapshot();
    });
    it("it should render a second-swipper-container" ,()=>{
        expect(container.find('div.card-detail-container')).toBeTruthy();
    });
});