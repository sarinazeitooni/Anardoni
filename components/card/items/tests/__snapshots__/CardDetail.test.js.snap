// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardDetail component it should render a div 1`] = `ShallowWrapper {}`;
